go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1365,6 +1367,19 @@ func DummyNextHopIPs() []net.IP {
 	return nextHops
 }
 
+// DummyMasqueradeIPs returns the fake host masquerade IPs used for service traffic routing.
+// It is used in: br-ex, where we SNAT the traffic destined towards a service IP
```

Comment:
It might be worth adding that these are only used for primary UDNs.

Reply:
`DummyMasqueradeIPs` is just a function that returns the IPs and it is not really specific to UDN. I have the explanation in `CreateDummyGWMacBindings`.